internal/git/gogit/git.go

```diff
@@ -75,7 +75,7 @@ func (g *Git) Changes() (bool, error) {
 }
 
 // Commit and push all changes
-func (g *Git) Commit(commitAuthor *domain.CommitAuthor, commitMessage string) error {
+func (g *Git) Commit(commitAuthor *git2.CommitAuthor, commitMessage string) error {
```

Comment:
Not really a fan of `git2` as a package name. Maybe `go-git` can be referred to as `gogit` and the new package can be `git`?

Reply:
That was an accident, it will be fixed.